### ISSUE TITLE
 HTTP Send: Invalid HTTP version parsed

### ISSUE THREAD

**Issue by quentinlesceller** [mimblewimble/grin-wallet#352]
Grin-wallet v3.1.0.
Getting this error (and also #349 which might be unrelated) when sending http payouts. Tested with Poloniex, Kucoin and Hotbit.
No errror with grin-wallet v3.0.0.
 
```
20200303 20:08:10.453 ERROR grin_wallet_impls::adapters::http - Performing version check (is recipient listening?): Request error: Cannot make request: invalid HTTP version parsed
```

**Comment by jaspervdm** [comment]
Seems like `hyper-timeout` does not play nice with the connector from `hyper_rustls`. I've filed an issue there: https://github.com/hjr3/hyper-timeout/issues/13

**Comment by jaspervdm** [comment]
Fixed in v3.1.1
